src/System.Management.Automation/engine/Modules/NewModuleManifestCommand.cs

```diff
@@ -484,6 +484,10 @@ private string QuoteName(object name)
         {
```

Comment:
The method comments should be updated to reflect string or URI.  Better would be to have two explicit overloads and remove the 'object' overload since I don't think anything other than uri and string are expected.